.github/workflows/build.yml

```diff
@@ -0,0 +1,213 @@
+name: Build and Test
+
+on:
+  push:
+    branches: master
+  pull_request:
+    branches: "*"
+
+jobs:
+  build:
+    name: Build tljh_repo2docker
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install Conda environment with Micromamba
+        uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-name: tljh
+          micromamba-version: "1.5.5-0"
+          init-shell: bash
+          create-args: >-
+            python=3.10
+            nodejs=18
+            yarn=3
+
+      - name: Setup pip cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: pip-3.10-${{ hashFiles('setup.py') }}
+          restore-keys: |
+            pip-3.10-
+            pip-
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Setup yarn cache
+        uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: yarn-${{ hashFiles('./yarn.lock') }}
+          restore-keys: |
+            yarn-
```

Comment:
These would be handled by `jupyterlab/maintainer-tools/.github/actions/base-setup@v1` (mentioned above).